Google silver sponsor announcement (#82)
trifectatechfoundation/trifectatech-website: content/blog/New Silver Sponsor Google.md

+++
title = "Trifecta Tech Foundation welcomes Google as Silver Sponsor"
slug = "new-silver-sponsor-google"
authors = ["Erik Jonkers"]
date = "2026-08-24"

[taxonomies]
tags = ["data-compression"] 

[extra]
image = "/blog/google-sponsor-announcement-trifecta-tech-foundation.png"

+++

Trifecta Tech Foundation is delighted to welcome Google as a new Silver sponsor! We're grateful for their €20,000/year sponsorship, a significant contribution to the flexible funding needed to sustain the maintenance work that keeps our critical infrastructure projects secure.

<!-- more -->

> The support comes from Google's Safe Coding team that works on replacing common C(++) libraries with Rust alternatives, at scale. Their work aligns with our goals of making critical infrastructure software safer by reducing attach surface through robust, inherently safer components.

Reviewer: ```suggestion
The support comes from Google's Safe Coding team, who work on replacing common C(++) libraries with Rust alternatives, at scale. Their work aligns with our goals of making critical infrastructure software safer by reducing attack surface through robust, inherently safer components.
```

> Google [has long advocated for a Safe Coding strategy](https://storage.googleapis.com/gweb-research2023-media/pubtools/7665.pdf) and is now evaluating the feasibility of [Scaling Memory Safety](/) through a pilot project for AI-assisted rewrites.

Reviewer: ```suggestion
Google [has long advocated for a Safe Coding strategy](https://storage.googleapis.com/gweb-research2023-media/pubtools/7665.pdf) (see also the recording of Bastian Kersting's [RustWeek 2026 talk](https://www.youtube.com/watch?v=mLzJfuaomZw) on this topic) and is now evaluating the feasibility of [Scaling Memory Safety](/) through a pilot project for AI-assisted rewrites.
```
